week-8/Homework/mandatory/3-dog-photo-gallery/index.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>DogPhotoShow </title>
+    <meta name="description" content="FetchExercise">
+    <meta name="author" content="BuchraAtkeh">
+    <style>
+        h1 {
+            border: 1px solid #ccc;
+            background-color: #ddd;
+            text-align: center;
+            padding: 30px;
+            color: rgb(128, 128, 202);
+        }
+        
+        #newDogPhoto {
+            display: block;
+            margin-left: auto;
+            margin-right: auto;
+            font-size: large;
+            color: rgb(106, 19, 206);
+            width: 200px;
+            height: 40px;
+            background-color: rgb(188, 196, 221);
+        }
```

Comment:
Simple and clear design! My suggestion is to add `:active` and `:hover` pseudo CSS classes to the button to make it more distinctive when the mouse cursor has hovered and the button is clicked. That could elevate the user experience.
https://paulund.co.uk/css3-buttons-with-pseudo-classes